.github/workflows/CI.yml

```diff
@@ -13,90 +13,59 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v1
-        with:
-          fetch-depth: 1
+        uses: actions/checkout@v2
       - name: Install Node.js
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v2
         with:
-          node: 12.x
+          node-version: 16
       - name: Install Packages
         run: npm install
       - name: Lint
         run: npm run -s lint
 
   test:
     name: Test
-
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest, macOS-latest]
-        eslint: [6.x, 5.x]
-        node: [13.x, 12.x, 10.x, 8.x]
-        exclude:
-          # On Windows, run tests with only the latest LTS environments.
-          - os: windows-latest
-            eslint: 6.x
-            node: 13.x
-          - os: windows-latest
-            eslint: 6.x
-            node: 10.x
-          - os: windows-latest
-            eslint: 6.x
-            node: 8.x
-          - os: windows-latest
-            eslint: 5.x
-            node: 13.x
-          - os: windows-latest
-            eslint: 5.x
-            node: 12.x
-          - os: windows-latest
-            eslint: 5.x
-            node: 10.x
-          - os: windows-latest
-            eslint: 5.x
-            node: 8.x
-          # On macOS, run tests with only the latest LTS environments.
-          - os: macOS-latest
-            eslint: 6.x
-            node: 13.x
-          - os: macOS-latest
-            eslint: 6.x
-            node: 10.x
-          - os: macOS-latest
-            eslint: 6.x
-            node: 8.x
-          - os: macOS-latest
-            eslint: 5.x
-            node: 13.x
-          - os: macOS-latest
-            eslint: 5.x
-            node: 12.x
-          - os: macOS-latest
-            eslint: 5.x
-            node: 10.x
-          - os: macOS-latest
-            eslint: 5.x
-            node: 8.x
-          # Run ESLint 5.x tests on only the latest LTS Node.
-          - os: ubuntu-latest
-            eslint: 5.x
-            node: 13.x
-          - os: ubuntu-latest
-            eslint: 5.x
-            node: 10.x
-          - os: ubuntu-latest
-            eslint: 5.x
-            node: 8.x
+        os: [ubuntu-latest]
+        eslint: ["^8.0.0-0"]
```

Comment:
Why not running it against ESLint v7 too?

Reply:
It's running against older ESLint versions too (see below)

https://github.com/mysticatea/eslint-plugin-node/pull/295/files#diff-3ab46ee209a127470fce3c2cf106b1a1dbadbb929a4b5b13656a4bc4ce19c0b8R48-R57